linkerd/app/outbound/src/lib.rs

```diff
@@ -66,7 +66,7 @@ impl Config {
         HttpEndpoint,
         Error = Error,
         Future = impl future::Future + Send,
-        Response = impl tokio::io::AsyncRead + tokio::io::AsyncWrite + Unpin + Send + 'static,
+        Response = impl io::AsyncRead + io::AsyncWrite + Unpin + Send + 'static,
```

Comment:
nit/tioli: for brevity's sake, i wonder if we want to start exposing the `Io` trait or something, so we don't have to write all this out...not a blocker.